src/backend/ingestion/index.ts

```diff
@@ -220,3 +240,121 @@ export class Ingestion extends Construct implements IGrantable {
     });
   }
 }
+
+interface ReprocessIngestionWorkflowProps {
+  readonly bucket: IBucket;
+  readonly queue: IQueue;
+}
+
+/**
+ * A StepFunctions State Machine to reprocess every currently indexed package
+ * through the ingestion function. This should not be frequently required, but
+ * can come in handy at times.
+ */
+class ReprocessIngestionWorkflow extends Construct {
```

Comment:
This description isn't really helpful to the reader. Perhaps we could just reference the runbook instead if the context isn't appropriate to include/maintain inline?
```suggestion
/**
 * A StepFunctions State Machine to reprocess every currently indexed package
 * through the ingestion function. This should not have to be run frequently
 * - see the runbook for more information. 
 */
class ReprocessIngestionWorkflow extends Construct {
```